From 5acf8980ba58aec010a5539b2bc6539c50567890 Mon Sep 17 00:00:00 2001
From: tos <tosiara@users.noreply.github.com>
Date: Tue, 24 Jun 2014 17:52:56 +0300
Subject: [PATCH 22/26] Remove some files that should not be in source tree
 config.h is auto generated upon configure, motion.mk808.conf was temporary
 experimental config from test branch - it's not needed in the scope of
 current fork

---
 config.h | 128 ---------------------------------------------------------------
 1 file changed, 128 deletions(-)
 delete mode 100644 config.h

diff --git a/config.h b/config.h
deleted file mode 100644
index c479b73..0000000
--- a/config.h
+++ /dev/null
@@ -1,128 +0,0 @@
-/* config.h.  Generated from config.h.in by configure.  */
-/* config.h.in.  Generated from configure.in by autoheader.  */
-
-/* Define to 1 if you have the <fcntl.h> header file. */
-#define HAVE_FCNTL_H 1
-
-/* Define to 1 if you have the `get_current_dir_name' function. */
-#define HAVE_GET_CURRENT_DIR_NAME 1
-
-/* Define to 1 if you have the <inttypes.h> header file. */
-#define HAVE_INTTYPES_H 1
-
-/* Define to 1 if you have the <linux/videodev2.h> header file. */
-#define HAVE_LINUX_VIDEODEV2_H 1
-
-/* Define to 1 if you have the <linux/videodev.h> header file. */
-/* #undef HAVE_LINUX_VIDEODEV_H */
-
-/* Define to 1 if you have the <memory.h> header file. */
-#define HAVE_MEMORY_H 1
-
-/* Define to 1 if you have MySQL support */
-/* #undef HAVE_MYSQL */
-
-/* Define to 1 if you have PostgreSQL support */
-/* #undef HAVE_PGSQL */
-
-/* Define to 1 if you have SDL support */
-#define HAVE_SDL 1
-
-/* Define to 1 if you have the <signal.h> header file. */
-#define HAVE_SIGNAL_H 1
-
-/* Define to 1 if you have SQLITE3 support */
-/* #undef HAVE_SQLITE3 */
-
-/* Define to 1 if you have the <stdint.h> header file. */
-#define HAVE_STDINT_H 1
-
-/* Define to 1 if you have the <stdio.h> header file. */
-#define HAVE_STDIO_H 1
-
-/* Define to 1 if you have the <stdlib.h> header file. */
-#define HAVE_STDLIB_H 1
-
-/* Define to 1 if you have the <strings.h> header file. */
-#define HAVE_STRINGS_H 1
-
-/* Define to 1 if you have the <string.h> header file. */
-#define HAVE_STRING_H 1
-
-/* Define to 1 if you have the <sys/ioctl.h> header file. */
-#define HAVE_SYS_IOCTL_H 1
-
-/* Define to 1 if you have the <sys/mman.h> header file. */
-#define HAVE_SYS_MMAN_H 1
-
-/* Define to 1 if you have the <sys/param.h> header file. */
-#define HAVE_SYS_PARAM_H 1
-
-/* Define to 1 if you have the <sys/stat.h> header file. */
-#define HAVE_SYS_STAT_H 1
-
-/* Define to 1 if you have the <sys/types.h> header file. */
-#define HAVE_SYS_TYPES_H 1
-
-/* Define to 1 if you have the <time.h> header file. */
-#define HAVE_TIME_H 1
-
-/* Define to 1 if you have the <unistd.h> header file. */
-#define HAVE_UNISTD_H 1
-
-/* Define to the address where bug reports for this package should be sent. */
-#define PACKAGE_BUGREPORT ""
-
-/* Define to the full name of this package. */
-#define PACKAGE_NAME "motion"
-
-/* Define to the full name and version of this package. */
-#define PACKAGE_STRING "motion trunkREVUNKNOWN"
-
-/* Define to the one symbol short name of this package. */
-#define PACKAGE_TARNAME "motion"
-
-/* Define to the home page for this package. */
-#define PACKAGE_URL ""
-
-/* Define to the version of this package. */
-#define PACKAGE_VERSION "trunkREVUNKNOWN"
-
-/* The size of `int', as computed by sizeof. */
-#define SIZEOF_INT 4
-
-/* The size of `int *', as computed by sizeof. */
-#define SIZEOF_INT_P 4
-
-/* The size of `long int', as computed by sizeof. */
-#define SIZEOF_LONG_INT 4
-
-/* The size of `long long', as computed by sizeof. */
-#define SIZEOF_LONG_LONG 8
-
-/* The size of `short', as computed by sizeof. */
-#define SIZEOF_SHORT 2
-
-/* The size of `void *', as computed by sizeof. */
-#define SIZEOF_VOID_P 4
-
-/* Define to 1 if you have the ANSI C header files. */
-#define STDC_HEADERS 1
-
-/* Define to empty if `const' does not conform to ANSI C. */
-/* #undef const */
-
-/* Define to 1 if you have av_avformat_alloc_context support */
-/* #undef have_av_avformat_alloc_context */
-
-/* Define to 1 if you have av_get_media_type_string support */
-#define have_av_get_media_type_string 1
-
-/* Define to 1 if you have av_register_protocol support */
-/* #undef have_av_register_protocol */
-
-/* Define to 1 if you have av_register_protocol2 support */
-/* #undef have_av_register_protocol2 */
-
-/* Define to 1 if you have avformat_alloc_context support */
-/* #undef have_avformat_alloc_context */
-- 
1.8.1.4

